### PR TITLE
Publish experimental results for host-guest systems

### PR DESCRIPTION
**Pull request by andrrizzi**
Here I have added the experimental results files for the binding host-guest challenge in `host_guest/Analysis/ExperimentalMeasurements/`.

### CODE REVIEW

File: host_guest/Analysis/ExperimentalMeasurements/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Manifesto
```
Comment (davidlmobley) [inline host_guest/Analysis/ExperimentalMeasurements/README.md]: Did you, er, mean "Manifest"? :)

**Comment by davidlmobley** [comment]
Also, once merged we should probably also send an e-mail announcement. Ping me and I will...? Or have you been sending e-mails?

File: host_guest/Analysis/ExperimentalMeasurements/README.md (hunk 1 above)
Comment (andrrizzi) [reply host_guest/Analysis/ExperimentalMeasurements/README.md]: Whoops! Yes, of course :)

**Comment by andrrizzi** [comment]
Done, thanks! Feel free to send the announcement email, @davidlmobley. I haven't sent any so far.

**Pull request merged**